### PR TITLE
implement apply button

### PR DESCRIPTION
**Pull request by judz5**
#14 
Apply button now opens webpage of link from `_var_internship_link` _(if valid)_

### CODE REVIEW

File: agtern/gui/frames/internship_detail_frame.py [hunk 1]
```diff
@@ -71,4 +71,9 @@ def _on_apply_now_bttn_click(self):
         button is clicked.
         """
         print('"Apply now" was clicked and the link is ' \
-              f'"{self._var_internship_link.get()}"')
\ No newline at end of file
+              f'"{self._var_internship_link.get()}"')
+
+        link = self._var_internship_link.get()
+        if(link): # Ensure not empty
```
Comment (johndpjr) [inline agtern/gui/frames/internship_detail_frame.py:77]:
```suggestion
        if link:  # Ensure not empty
```
Complying with PEP8 standards: at least 2 spaces for inline comments and no parentheses in `if` statement

File: agtern/gui/frames/internship_detail_frame.py [hunk 2]
```diff
@@ -71,4 +71,9 @@ def _on_apply_now_bttn_click(self):
...
         """
         print('"Apply now" was clicked and the link is ' \
-              f'"{self._var_internship_link.get()}"')
\ No newline at end of file
+              f'"{self._var_internship_link.get()}"')
+
+        link = self._var_internship_link.get()
+        if(link): # Ensure not empty 
+            webbrowser.open_new_tab(link)
+
```
Comment (johndpjr) [inline agtern/gui/frames/internship_detail_frame.py:79]:
```suggestion
```
Extra line

**Review by johndpjr — CHANGES_REQUESTED** [review]
In addition to a few formatting things, I know I didn't add this on the TODO, but please modify the docstring of this function to describe what the function does (what is the "Response" of the callback?)

**Comment by judz5** [comment]
👍🏻 let me know if you want more detail in docstring, hopefully formatting is okay now.

**Comment by johndpjr** [comment]
Other than the formatting suggestions, looks good!

**Comment by judz5** [comment]
which formatting suggestions? was there more other than the spacing and parentheses you said?

**Comment by judz5** [comment]
oops, my bad i thought i had no parentheses and you wanted them lol ill fix that

**Comment by johndpjr** [comment]
One more change - there's an "s" after `self,`. Check my suggested change

**Comment by judz5** [comment]
ok hopefully thats it, sorry for all the little mistakes

**Review by johndpjr — APPROVED** [review]
Looks good!

**Pull request merged**